### DISCUSSION TITLE
What is the best OR Regular expression style?

### DISCUSSION THREAD

**Discussion by noraj** [tldr-pages/tldr#14298] [Polls]
Example for _save and quit_ in `nvim` and `vim`, you could use:

- <kbd>\<Esc></kbd> + <kbd>Z</kbd> + <kbd>Z</kbd>
- OR <kbd>\<Esc></kbd> + <kbd>:</kbd> + <kbd>x</kbd>
- OR <kbd>\<Esc></kbd> + <kbd>:</kbd> + <kbd>w</kbd> + <kbd>q</kbd>

**Comment by kbdharun** [comment]
Completed with https://github.com/tldr-pages/tldr/pull/14297

**Reply by noraj** [reply]
In the guidelines I didn't see any rule that could answer this question about the preferred style.

**Reply by kbdharun** [reply]
> In the guidelines I didn't see any rule that could answer this question about the preferred style.

Yeah, currently we only document this syntax for [options](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax), I will add the current proposal to the [formatting](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#formatting) section.

**Comment by sbrl** [comment]
I would suggest here that a poll is not the best way to go about this, @noraj. Generally speaking here in tldr-pages we prefer to make collective decisions by discussing and coming to an agreement, rather than just deciding based on the popular vote.

This enables more voices to be heard and a better decision to be ultimately reached, even if progress can seem slow sometimes.
